infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServerCmdLine.java

```diff
@@ -46,6 +46,7 @@ private CorfuServerCmdLine() {
                     + "[" + HEALTH_PORT_PARAM + "=<health_port>]"
                     + "[--snapshot-batch=<batch-size>] [--lock-lease=<lease-duration>]"
                     + "[--max-snapshot-entries-applied=<max-snapshot-entries-applied>]"
+                    + "[--corfu-port-for-lr=<corfu-port-for-lr>]"
```

Comment:
Just a minor comment, wouldn't `corfu-port` be enough? In the description you can add that it's used for LR. In any case this port is not specific for LR, it's Corfu's port.

Reply:
I see the point but these command line params are also shared with the CorfuServer, which already has a mandatory parameter for <port>.  So adding a new parameter named corfu-port can be confusing.

Reply:
So if it's already there and mandatory, why not rely on that same parameter? Less error prone.

Reply:
We cannot use that parameter.  The way it works is as follows:

CorfuServer [optional params] <port>

The same command is used for starting LR and Corfu Server.  For LR, the <port> argument is set to the LR port, i.e., 9010/20 etc.  For Corfu, it is set to 9000.  We do not get the Corfu port through that param.

Reply:
Ideally the existing parameter should be named just `port` and then the new one can be called `corfu-port`

Reply:
+1 on Sundar's suggestion, anyways seems from Pankti's snippet that the existing is already called 'port'